src/BenchmarkDotNet/Columns/SizeUnit.cs

```diff
@@ -35,7 +35,16 @@ public static SizeUnit GetBestSizeUnit(params long[] values)
             if (!values.Any())
                 return B;
             // Use the largest unit to display the smallest recorded measurement without loss of precision.
-            long minValue = values.Min();
+            long minValue = values.Min();
+            long maxValue = values.Max();
+
+            const long maxSmallBytes = 9 * 1024;
```

Comment:
Why not `(10 * 1024) - 1`?

Reply:
There is no special reason. Even 9 KB seemed too much to me. But I have no strong opinion about that.

Reply:
Maybe a better solution would be to print fractional values if it's not measures in bytes.

Reply:
I'm not sure, if I understand right.
I'm new in this codebase, I thought the result is produced by https://github.com/dotnet/BenchmarkDotNet/blob/602d3da11524f842a581183f3fe62c92910c5519/tests/BenchmarkDotNet.Tests/SizeUnitTests.cs#L47 and this method already produce fractional numbers https://github.com/dotnet/BenchmarkDotNet/blob/602d3da11524f842a581183f3fe62c92910c5519/tests/BenchmarkDotNet.Tests/SizeUnitTests.cs#L39

Did I miss something or do you mean something else?

Reply:
I haven't looked at this particularly deeply myself, but the issue that this is trying to fix mentions that for 1KB to 2KB, it does not use fractional. So maybe there is a bug in there that should be reporting fractional that needs to be fixed.